src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -2367,4 +2410,16 @@ registerThemingParticipant((theme, collector) => {
 			collector.addRule(makeTabBackgroundRule(adjustedColor, adjustedColorDrag, false, false));
 		}
 	}
+
+	const tabDndIndicatorColor = theme.getColor(TAB_DRAG_AND_DROP_BETWEEEN_INDICATOR);
```

Comment:
Is there a reason this CSS rule cannot be in the CSS file? We try to keep dynamic registrations at a minimum using CSS variables.